packages/neo-one-smart-contract-lib/src/ownership/Ownable.ts

```diff
@@ -0,0 +1,37 @@
+import { Address, constant, createEventNotifier, SmartContract } from '@neo-one/smart-contract';
+
+const transferredOwnership = createEventNotifier<Address | undefined, Address>('transferred_ownership', 'from', 'to');
+
+export function Ownable<TBase extends Constructor<SmartContract>>(base: TBase) {
+  abstract class OwnableClass extends base {
+    protected mutableOwner: Address | undefined;
```

Comment:
Make this `private`

Reply:
Doing so prevents:
```  public constructor(protected mutableOwner: Address = Deploy.senderAddress) {```
Unless it is optimal to take in owner and have the extending class responsible for `this.initialOwner(owner)` that has access to set the private `mutableOwner` member.

Reply:
Yea I think that's the right way to go about it - we don't want subclasses arbitrarily modifying this value.

Reply:
👍